src/panels/menu/modal/SaveDiagramsModal.tsx

```diff
@@ -125,6 +126,30 @@ export const SaveDiagramsModal: React.FC<Props> = (props) => {
     svg.setAttribute("width", String(area.width));
     svg.setAttribute("height", String(area.height));
     svg.setAttribute("viewbox", `0 0 ${area.width} ${area.height}`);
+    if (format === "SVG" && iriData) {
+      svg.setAttribute(
+        "data-ontographer-view",
+        Representation[diagramRepresentation].toLowerCase()
+      );
+      const termElements = svg.getElementsByClassName("joint-element");
```

Comment:
These two code blocks seem to be easily extractable to a single method with two parameters - might make the code more readable.

Otherwise looks OK IMO. 

Cannot test the functionality however.